bench/workgen/workgen.cxx

```diff
@@ -118,6 +117,43 @@ static void *thread_runner_main(void *arg) {
     return (NULL);
 }
 
+static void *thread_workload(void *arg) {
+
+    Workload *runner = (Workload *) arg;
+
+    try {
+        runner->increment_timestamp();
+    } catch (WorkgenException &wge) {
+        std::cerr << "Exception while incrementing timestamp." << std::endl;
+    }
+
+    return (NULL);
+}
+
+int Workload::increment_timestamp() {
+    char buf[500];
+    uint64_t time_us;
+
+    while(!stop_timestamp_thread)
```

Comment:
Use space after `while` or `if` and before `(`.

Reply:
Fixed